turborepo-tests/integration/tests/run-logging/errors-only.t

```diff
@@ -48,8 +48,8 @@ Setup
   app-a:builderror: npm ERR! Error: command failed 
   app-a:builderror: npm ERR!   in workspace: app-a 
   app-a:builderror: npm ERR!   at location: .* (re)
-  app-a:builderror: ERROR: command finished with error: command .* npm run builderror exited \(1\) (re)
```

Comment:
curious what's up with this change?

Reply:
Due to how Rust finds binaries to run we need to first run `which` on package manager binaries in order to find them on Windows. Outcome of this is that our process warnings have the entire path to the binary instead of just the binary name.